Source/DafnyCore/Verifier/BoogieGenerator.ExpressionWellformed.cs

```diff
@@ -786,10 +822,16 @@ private record ReadsCheckDelayer(ExpressionTranslator etran, Function selfCallsA
                 }
                 if (wfOptions.DoReadsChecks) {
                   // check that the callee reads only what the caller is already allowed to read
-                  var s = new Substituter(null, new Dictionary<IVariable, Expression>(), e.GetTypeArgumentSubstitutions());
+
+                  // substitute actual args for parameter is frames for the description expression...
```

Comment:
Not critical, but this comment isn't super clear to me. Maybe it could be reworded?

Reply:
Oops, there was a typo (and generally unclear wording). I've fixed this in the latest commit.